crates/types/src/identifiers.rs

```diff
@@ -632,7 +643,7 @@ pub struct IdempotencyId {
 impl IdempotencyId {
     pub fn new(
         service_name: ByteString,
-        service_key: Option<Bytes>,
+        service_key: Option<ByteString>,
```

Comment:
TODO for myself: verify if this affects queries to idempotency table in rocksdb.

Reply:
Checked, this looks fine.